### PR TITLE
Add MFH Adapter to iaf-test

### PR DESCRIPTION
**Pull request by alisihab**

### CODE REVIEW

File: test/src/main/configurations/FileHandling/Configuration_MFH.xml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+			<XsltPipe
+				name="wrapInput"
+				styleSheetName="MFH/xsl/createTriggerMessage.xsl"
+				storeResultInSessionKey="triggerMessage"
+			>
+				<Param name="CanonicalName" xpathExpression="file/@canonicalName" />
+				<Param name="Filename" xpathExpression="file/@name" />
+				<Param name="MFHFlowId" value="${mfh.flow1.mfhflowid}" /> 
+				<Param name="FlowId" value="${mfh.flow1.flowid}" /> <!-- Can be hardcoded or retrieved from the filename depending on the new design-->
```
[the remark this answers is not in the document]
Comment (gvanbrakel) [reply test/src/main/configurations/FileHandling/Configuration_MFH.xml:35]: If one of the two is there only to support legacy-fxf, then that one should have the long name, e.g. FxfFlowId, and the new one should be called FlowId
Could you also provide a confguration without the legacy?

**Review by gvanbrakel — CHANGES_REQUESTED** [review]

File: test/src/main/configurations/FileHandling/DeploymentSpecifics.properties [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+active.mfh=true
+
+# the folder the directoryListener listens to
+mfh.flow1.rootdir=${testdata.dir}/mfh
+# to make sure the file is no longer being written by another process
+mfh.flow1.minstabletime=1001
+# directory listener filters files by name 
+mfh.flow1.wildcard=mfh-*
+# legacy flow id
+mfh.flow1.flowid=NNX12345
```
Comment (gvanbrakel) [inline test/src/main/configurations/FileHandling/DeploymentSpecifics.properties:12]: Could you please rename this one to mfh.flow1.legacyflowid, too?

**Review by gvanbrakel — CHANGES_REQUESTED** [review]

**Review by gvanbrakel — APPROVED** [review]

File: test/src/test/testtool/ManagedFileHandler/scenario01.properties [hunk 3]
```diff
@@ -0,0 +1,18 @@
+scenario.description = MFH process one xml file
+
+include = common.properties
+
+dir.init.filename						= scenario01/dir-init.xml
+
+step1.jdbc.deleteRecords.read			= common/rowsupdated-dummy.xml
+
+step2.dir.init.write					= common/dummy.txt
```
Comment (nielsm5) [inline test/src/test/testtool/ManagedFileHandler/scenario01.properties:9]: you can also use `writeLine` that way you don't have to create a new (empty) file

File: test/src/main/configurations/FileHandling/MFH/xsl/createTriggerMessage.xsl [hunk 4]
```diff
@@ -0,0 +1,31 @@
+<xsl:stylesheet version="2.0"
+	xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+	<xsl:output method="xml" indent="no" omit-xml-declaration="yes" />
+	
+	<xsl:param name="CanonicalName"/>
+	<xsl:param name="Filename"/>
+	<xsl:param name="MFHFlowId"/>
+	<xsl:param name="LegacyFlowId"/>
+	<xsl:param name="LocalTransactionID"/>
+	<xsl:param name="originalFilename"/>
```
Comment (nielsm5) [inline test/src/main/configurations/FileHandling/MFH/xsl/createTriggerMessage.xsl:10]: originalFilename doesn't start with a capital o but the other params do!

**Review by nielsm5 — APPROVED** [review]

**Pull request merged**